src/ngraph/runtime/cpu/CMakeLists.txt

```diff
@@ -136,7 +145,11 @@ if (NGRAPH_CPU_ENABLE)
         target_link_libraries(cpu_backend PRIVATE ${MPI_CXX_LIBRARIES})
     endif()
 
-    target_link_libraries(cpu_backend PUBLIC ngraph codegen libmkldnn libeigen libjson libtbb)
+    add_dependencies(cpu_backend ext_mkldnn ext_eigen)
```

Comment:
This add_dependencies should not be needed because the target_link_libraries already depends on libmkldnn and libeigen. I recall that there is a problem with libmkldnn where it does not depend on ext_mkldnn and it should depend on that, but that would be a change in external_mkldnn.cmake.